src/components/SchemaRow/useChoices.ts

```diff
@@ -12,12 +12,11 @@ type Choice = {
 };
 
 function calculateChoiceTitle(node: SchemaNode, isPlural: boolean): string {
-  const complexObjectSuffix = isPlural ? '-s' : '';
   const primitiveSuffix = isPlural ? 's' : '';
```

Comment:
Let's get rid of this too? puralization is hard in English and putting an s on the end of things doesn't always work.

Oppertunitys
Monkeys

Doesn't work. :)

Reply:
That second one does work AFAICT. It's for primitives, I tried them all, they all luckily work. *objects*, *integers*, *booleans*, *nulls*, *strings*.

I believe it feels much more natural to have that pluralization there, but LMK if you can reproduce a broken one.